src/patternfly/components/DualListSelector/dual-list-selector.scss

```diff
@@ -226,6 +225,10 @@ $pf-v5-c-dual-list-selector__item--MaxNesting: 10;
 
       font-weight: var(--#{$dual-list-selector}__list-item-row--m-selected__text--FontWeight);
     }
+
+    &:hover {
+      --#{$dual-list-selector}__list-item-row--BackgroundColor: var(--#{$dual-list-selector}__list-item-row--hover--BackgroundColor);
+    }
```

Comment:
Might be a duplicate of the style on line 216

Reply:
It's needed here as well for `pf-m-selected` rows to also get hover styling.